user/pages/02.development/05.troubleshooting.notes/01.audio.issues/audio.md

```diff
@@ -0,0 +1,98 @@
+
+# Audio errors and issues
+If your questions and issues aren't addressed here please reach out to us via the support channels.
+
+## Alsa
+Alsa is the sound architecture on most linux systems but particularly on Ubuntu and Raspbian. Alsa comes with a series of tools used to select a usb audio device and configure them for use with your device.
```

Comment:
Add a comma between `linux systems` and `but`, and replace `Alsa comes with...` with `It comes with...`.